plugins/org.eclipse.n4js.model/emf-gen/org/eclipse/n4js/n4JS/impl/N4JSPackageImpl.java

```diff
@@ -1583,6 +1583,11 @@ public static N4JSPackage init() {
 
 		isInited = true;
 
+		// Initialize simple dependencies
```

Comment:
In few places we explicitly trigger initialization of dependencies (e.g. `org.eclipse.n4js.N4JSStandaloneSetup.createInjectorAndDoEMFRegistration()`). With this code being generated now, do we still need to do it? If `no` then we should refactor those setup files.